upf/pfcp.c

```diff
@@ -5099,6 +5104,10 @@ static struct pfcp_group_ie_def pfcp_create_urr_group[] =
       .type = PFCP_IE_QUOTA_HOLDING_TIME,
       .offset = offsetof(pfcp_create_urr_t, quota_holding_time)
     },
+    [CREATE_URR_QUOTA_VALIDITY_TIME] = {
+      .type = PFCP_IE_QUOTA_VALIDITY_TIME,
+      .offset = offsetof(pfcp_create_urr_t, quota_validity_time)
+    },
```

Comment:
wrong order, Quota Validity Time should go after Dropped DL Traffic Threshold to keep it in sync with TS 29.244